src/main/java/uk/gov/pay/api/service/ConnectorUriGenerator.java

```diff
@@ -35,6 +35,11 @@ public String chargesURI(Account account, String agreementId) {
     public String chargesURIWithParams(Account account, Map<String, String> queryParams) {
         return buildConnectorUri(account, format("/v1/api/accounts/%s/charges", account.getAccountId()), queryParams);
     }
+    
+    public String refundsURIWithParams(Account account, Map<String, String> queryParams) {
+        String path = format("/v1/accounts/%s/refunds", account.getAccountId());
```

Comment:
This does not match the endpoint in connector :(

Reply:
Updated! Thanks